apps/src/code-studio/progress.js

```diff
@@ -141,9 +196,14 @@ progress.renderCourseProgress = function (scriptData, currentLevelId) {
 /**
  * Initializes our redux store with initial progress
  * @param {object} store - Our redux store
- * @param scriptData
- * @param currentLevelId
- * @param saveAnswersBeforeNavigation
+ * @param {object} scriptData
+ * @param {string} scriptData.name
+ * @param {boolean} scriptData.disablePostMilestone
+ * @param {boolean} [scriptData.plc]
+ * @param {object[]} [scriptData.stages]
+ * @param {boolean} [scriptData.peerReviewsRequired]
+ * @param {string} currentLevelId
+ * @param {boolean} [saveAnswersBeforeNavigation]
```

Comment:
I had that at first, but although we have a parameter called scriptData in a couple of places, it's contents are not always the same (though it always contains data about the script). I decided it was better to just document the options being passed in in each case.